projects/dbus-broker/Dockerfile

```diff
@@ -14,7 +14,7 @@
 #
 ################################################################################
 
-FROM gcr.io/oss-fuzz-base/base-builder-rust
+FROM gcr.io/oss-fuzz-base/base-builder-rust:ubuntu-24-04
```

Comment:
dbus-broker can't be bumped due to https://github.com/google/oss-fuzz/issues/14220.